src/content/notebookContent.ts

```diff
@@ -59,7 +59,7 @@ export function addCellToNotebookContent(
   const id = data.id || generateUniqueCellId();
   const cell: Cell = {
     cellType,
-    textContent: "",
+    textContent: data.textContent ? data.textContent + "" : "",
```

Comment:
Why the `+ ""`? This gives me Java vibes, do you want to coerce it into a string? I propose to trust that it's a string, or to actually check it `typeof x === "string"`. 

Nit/note: this could be `data.textContent || ""`

Reply:
Good point, changed it.